src/components/Examinations.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable class-methods-use-this */
+import React from "react";
+import Navbar from "./Navbar";
+
+export default class Examinations extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2>History list of Stations</h2>
```

Comment:
Could add specific placeholder text on each component, so we can be sure the page is rendering the way we expect.

Reply:
done!